Docs/topics/bunkum/autodiscover-api.md

```diff
@@ -34,14 +37,18 @@ Date: Thu, 12 Oct 2023 00:47:58 GMT
 
 The server MUST reply with JSON data.
 
-The client can also specify a titleId in the request:
+The client MAY specify a `titleId` as a query parameter to identify the game in the request:
 
 `GET /autodiscover?titleId=NPUA80662 HTTP/1.1`
 
-If the server wants to, it can change the output of the `url` parameter based on the given titleId. For example a LBP3 client may require HTTPS, but other clients may not be able to connect via HTTPS. In this case you would want to change to a http(s) URL for those clients.
+If the server chooses to, it MAY change the output of the `url` parameter based on the given `titleId`. For example a LBP3 client may require HTTPS, but other clients may not be able to connect via HTTPS. In this case you would want to change to a http(s) URL for those clients.
```

Comment:
```suggestion
If the server chooses to, it MAY change the output of the `url` parameter based on the given `titleId`. For example an LBP3 client may require HTTPS, but other clients may not be able to connect via HTTPS. In this case you would want to change to an http(s) URL for those clients.
```